src/Types.fs

```diff
@@ -4,9 +4,15 @@ open GraphQLParser.AST
 
 [<RequireQualifiedAccess>]
 type OutputTarget =
-    | Fable
-    | FSharp
-    | Shared
+    | Fable = 0
+    | FSharp = 1
+    | Shared = 2
+
+[<RequireQualifiedAccess>]
+type SerializerType =
+    | System = 0
+    | Newtonsoft = 1
+    | Unknown = 2
```

Comment:
`Unknown` shouldn't be an option here. Default is `Newtonsoft`, the other option should be `SystemTextJson`

Reply:
Why not move to System as default? Do you have any specific reason to stay with Newtonsoft?

Reply:
The converter used from `Fable.Remoting.Json` has added features specifically to be able handle edge cases of GraphQL types such as unions and interfaces as I have mentioned before (see my comment [here](https://github.com/Zaid-Ajaj/Snowflaqe/pull/40#issuecomment-823187772))

The library has been battle-tested in production for years now which is enough reason for me to keep it as default.

Reply:
OK